src/review-form.ts

```diff
@@ -37,41 +41,49 @@ export class ReviewForm extends LitElement {
   @property({ type: Object }) oldReview?: Review;
 
   /* Errors to add to the form, if any */
-  @property({ type: Array }) errors?: string[];
+  @property({ type: Array }) errors: string[] = [];
+
+  /* Service for activating the recaptcha challenge */
+  @property({ type: Object }) recaptchaManager?: RecaptchaManagerInterface;
+
+  /* Recaptcha widget for the form */
+  @property({ type: Object }) recaptchaWidget?: RecaptchaWidgetInterface;
```

Comment:
This can probably be `@state() private` instead of `@property()` since you're not passing in the widget from outside of this component